mount/unmount_unix.go

```diff
@@ -2,13 +2,49 @@
 
 package mount
 
-import "golang.org/x/sys/unix"
+import (
+	"os/exec"
+
+	"golang.org/x/sys/unix"
+)
 
 func unmountBare(target string, flags int) error {
 	return unix.Unmount(target, flags)
 }
 
+// fuseSuperMagic is defined in statfs(2)
+const fuseSuperMagic = 0x65735546
```

Comment:
This needs to be added to `golang.org/x/sys/unix`...

Reply:
Good one, yes